### PR TITLE
iOS Admin app structure, Events list page, Events edit page

### PR DESCRIPTION
**Pull request by zuhairp**
screen shot 2017-03-28 at 12 19 28 am
screen shot 2017-03-28 at 12 21 03 am

### CODE REVIEW

File: mobile/Admin/Admin.iOS/Admin.iOS.csproj [hunk 1]
```diff
@@ -23,8 +22,6 @@
     <MtouchDebug>true</MtouchDebug>
     <MtouchFastDev>true</MtouchFastDev>
     <MtouchProfiling>true</MtouchProfiling>
-    <MtouchUseSGen>true</MtouchUseSGen>
```
Comment (Saadnajmi) [inline mobile/Admin/Admin.iOS/Admin.iOS.csproj]: Why are these removed?

File: mobile/Common/Common.Common/API/Entities/Event.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
+﻿using System;
```
Comment (Saadnajmi) [inline mobile/Common/Common.Common/API/Entities/Event.cs]: Why did you change the namespace?

Comment (zuhairp) [inline mobile/Common/Common.Common/API/Entities/Event.cs]: This is a separate Event model that aligns with admin API inputs

File: mobile/Admin/Admin.iOS/Admin.iOS.csproj (hunk 1 above)
Comment (zuhairp) [inline mobile/Admin/Admin.iOS/Admin.iOS.csproj]: No idea what those are

Comment (Saadnajmi) [inline mobile/Admin/Admin.iOS/Admin.iOS.csproj]: I'll just make sure it builds, if my VS decides to work today

Comment (zuhairp) [inline mobile/Admin/Admin.iOS/Admin.iOS.csproj]: Cool, shouldn't affect you though since it's iOS
